frontend/src/components/History/index.jsx

```diff
@@ -236,8 +241,17 @@ class History extends React.Component {
                     <strong>TIP:</strong> In this table you may edit metadata fields. Click on the value in the column and row where you want to change the data and it will open as a text box.
                 </p>
                 <CodebookLabelMenuContainer />
+                <div>
+                    <p style={{ maxWidth: "75ch" }}>
+                        Toggle the checkbox below to show/hide unlabeled data:
+                    </p>
+                    <label>
+                        <input type="checkbox" onChange={() => this.toggleShowUnlabeled()} />
+                        <span style={{ marginLeft: "4px" }}>Unlabled Data</span>
+                    </label>
+                </div>
                 <ReactTable
-                    data={history_data}
+                    data={this.state.showUnlabeled ? [...history_data, ...unlabeled_data] : history_data}
```

Comment:
See above. I think it would make sense to just have unlabeled be an argument to the API call and history data includes it or leaves it out based on that